alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/capabilities/BasicMovementCapability.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2010-2022, Danilo Pianini and contributors
+ * listed, for each module, in the respective subproject's build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.model.implementations.capabilities
+
+import it.unibo.alchemist.model.interfaces.capabilities.MovementCapability
+import it.unibo.alchemist.model.interfaces.capabilities.RunningCapability
+import it.unibo.alchemist.model.interfaces.capabilities.WalkingCapability
+
+/**
+ * Implementation of a basic [MovementCapability]
+ */
+class BasicMovementCapability :
+    MovementCapability,
+    WalkingCapability by BasicWalkingCapability(),
+    RunningCapability by BasicRunningCapability()
```

Comment:
```suggestion
    RunningCapability by BasicRunningCapability(),
```

Reply:
Actually I've pushed a refactoring of this...
I couldn't figure out how to properly use delegation and at the same time, 
give the possibility to specify custom speeds in the constructor, when instantiating